modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/launcher/Terminator.java

```diff
@@ -1,114 +1,9 @@
-/*
```

Comment:
fixed this in a later PR https://github.com/ballerinalang/composer/blob/master/modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/launcher/Terminator.java